third_party/move/move-binary-format/src/file_format.rs

```diff
@@ -850,28 +839,23 @@ impl AccessSpecifier {
 )]
 pub enum AccessKind {
     Reads,
-    Writes,
-    Acquires, // reads or writes
+    Writes, // Includes reading
 }
 
 impl AccessKind {
     /// Returns true if this access kind subsumes the other.
     pub fn subsumes(&self, other: &Self) -> bool {
         use AccessKind::*;
-        match (self, other) {
-            (Acquires, _) => true,
-            (_, Acquires) => false,
-            _ => self == other,
-        }
+        matches!((self, other), (_, Reads) | (Writes, Writes))
```

Comment:
Also here there is a semantic change: can we document the properties of reads and writes? 

nit: instead of `_` use `Reads | Writes` in case something gets added later